es/Algoritmos de selección/Selección Rápida.md

```diff
@@ -0,0 +1,50 @@
+# Selección rápida o Quick Select
+
+### Definición del problema
```

Comment:
```suggestion
## Definición del problema
```

Reply:
I agree with you here 100%. There are some other suggestions that you're pointing out but that were already there in the original file merges in english. Maybe that should be fixed also. Let me know what do I have to do in that case.